in_toto/settings.py

```diff
@@ -65,6 +65,8 @@
 # 'pyca-cryptography']
 GENERAL_CRYPTO_LIBRARY = 'pycrypto'
 
+# FIXME: Add as command line argument or to config file, e.g .in-toto-ignore
+ARTIFACT_EXCLUDES=["*.link*", ".git"]
```

Comment:
I like these defaults. I wonder if we can extend this list with `*.pyc` and  `*~`